README.md

```diff
@@ -45,6 +77,12 @@ $ cargo build
 
 ## Usage
 
+Run the following generate certs in the default directory `/etc/bpfd/certs/`:
+
+```shell
+sudo ./scripts/certificates.sh init
+```
+
 Load a sample XDP Program:
```

Comment:
In the examples just below here, the `./target/debug/bpfctl` command failed for me.
```
$ ./target/debug/bpfctl --help
18:01:28 [WARN] bpfctl::config: [bpfctl/src/config.rs:38] No config file provided. Using default
Error: CA Cert File does not exist

Caused by:
    Permission denied (os error 13)
```

If I run with `sudo` it works. I guess `sudo` needs to be added to the `bpfctl` command examples, or need to change permissions on the certs files.

Also, is there a clean way to stop `bpfd`? I ran `ps -ef | grep "bpfd" ` and `sudo kill 158009`, but wonder if there is a better way.

Reply:
Yes use `sudo` as the certs are owned by `root:root`.
Eventually they should be `bpfd:bpfd` and anyone in the `bpfd` group should be able to use `bpfctl` tool, but that's for another day.

In one terminal I run:
```
sudo ./target/debug/bpfd
```

I use `bpfctl` in another .... and then I `CTRL+C` on `bpfd` to stop it.